safe/gui/widgets/dock.py

```diff
@@ -1554,10 +1559,45 @@ def show_no_keywords_message(self):
             'Layer keywords missing:'), **WARNING_STYLE))
         context = m.Paragraph(
             self.tr(
-                'No keywords have been defined for this layer yet. If '
-                'you wish to use it as an impact or hazard layer in a '
-                'scenario, please use the keyword editor. You can open '
-                'the keyword editor by clicking on the '),
+                'No keywords have been defined for this layer yet. If you '
+                'wish to use it as exposure, hazard, or aggregation layer in '
+                'a scenario, please use the keyword wizard. You can open '
+                'the keyword wizard by clicking on the '),
+            m.Image(
+                'file:///%s/img/icons/'
+                'show-keyword-wizard.svg' % resources_path(),
+                **SMALL_ICON_STYLE),
+            self.tr(
+                ' icon in the toolbar.'))
+        report.add(context)
+        self.pbnPrint.setEnabled(False)
+        # noinspection PyTypeChecker
+        self.show_static_message(report)
+
+    def show_keyword_version_message(
+            self, keyword_version, inasafe_version):
+        """Show a message indicating that the keywords version is mismatch
+
+        :param keyword_version: The version of the layer's keywords
+        :type keyword_version: str
+
+        :param inasafe_version: The version of the InaSAFE
+        :type inasafe_version: str
+
+        .. note:: The print button will be disabled if this method is called.
+        """
+        LOGGER.debug('Showing Mismatch Version Message')
+        report = m.Message()
+        report.add(LOGO_ELEMENT)
+        report.add(m.Heading(self.tr(
+            'Layer Keyword\'s Version Mismatch:'), **WARNING_STYLE))
+        context = m.Paragraph(
```

Comment:
You layer\'s keyword version (%s) does not match your InaSAFE version (%s). If you wish to use this layer as an exposure, hazard or aggregation layer in an analysis, please use the keyword wizard to update the keywords. You can open the wizard by clicking on the